rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.stats/90_global_ordinals.yml

```diff
@@ -0,0 +1,242 @@
+---
+setup:
+  - do:
+      indices.create:
+        index:  test1
+        body:
+          settings:
+            index.number_of_shards: 1
+            index.number_of_replicas: 0
+          mappings:
+            properties:
+              bar:
+                type: keyword
+              baz:
+                type: keyword
+
+  - do:
+      index:
+        index: test1
+        id:    "1"
+        body:  { "bar": "bar", "baz": "baz" }
+
+  - do:
+      index:
+        index: test1
+        id:    "2"
+        body:  { "bar": "foo", "baz": "foo" }
+
+  - do:
+      indices.refresh: {}
+
+  # Enforce creating an extra segment in at least one shard,
+  # otherwise no global ordinals will be built.
+  - do:
+      index:
+        index: test1
+        id: "2"
+        body: { "bar": "foo", "baz": "foo" }
+
+  - do:
+      indices.refresh: {}
+
+  - do:
+      search:
```

Comment:
Move this to a test?

Reply:
This isn't really a test. It is needed for the indices stats to be able to report global ordinals. In this case they are computed lazily when needed by a search with aggs.